### DISCUSSION TITLE
posts/mikrotik-hap-ax3-wifi-config/

### DISCUSSION THREAD

**Discussion by giscus** [DarrenOfficial/DarrenOfficial#14] [General]
# posts/mikrotik-hap-ax3-wifi-config/

While Mikrotik wifi is nothing to write home about, it is still pretty usable. Do not play with frequencies and cipher suites, unless you know what are you doing. Specifically frequencies and band widths are pretty joined together, they have to fit each other. Your's don't. So unless you have a specific need, use defaults.

https://blog.darrennathanael.com/posts/mikrotik-hap-ax3-wifi-config/

**Comment by teddytodo** [comment]
Hello Daren,

I also have a MIkrotik hap ax3. I was wondering which RouterOS version you used to create multiple SSIDs with its own vlan. I searched the internet for info/documentation about this, but sadly enough the info I found is not working with the Mikrotik. With the default wifi-qcom package installed on the Mikrotik (when I bought it) the Wifi tab in the menu, there is no Virtual ap available when I press the + button. 
If I install the wirless package (on RouterOS 7.16.1) the Wireless tab is visible in the menu and you can select Virtual app. But I configure the Virtual app I the master interface option is empty. I cannot select an interface.
I also downgrade the Mikrotik to version 7.12.2 and installed the wifiwave2 package. But sadly enough I still cannot create a Virtual app.
Do you know how to achieve this on a Mikrotik hap ax3? Thanks in advance.

Teddy

**Reply by DarrenOfficial** [reply]
I'm now using the latest; You just need to pass the VLAN inside the Interface > DataPath, then enter in your VLAN ID.

I recommend using the latest RoS then adding a "slave" interface with the master set to the radio freq you'd want to use.

**Reply by DarrenOfficial** [reply]
![image](https://github.com/user-attachments/assets/6f33f31e-2ddb-4fe3-8bea-927d87fe6284)
